data/yam/agama/auto/template.libsonnet

```diff
@@ -15,6 +15,7 @@ function(bootloader=false,
          files=false,
          iscsi=false,
          localization='',
+         only_required=false,
```

Comment:
onl_required what? :) it would be better preceeding it with software, ie:
`software_only_required=false`